src/XR/features/WebXRHandTracking.ts

```diff
@@ -553,9 +556,22 @@ export class WebXRHandTracking extends WebXRAbstractFeature {
     private static _generateDefaultHandMeshesAsync(scene: Scene, options?: IWebXRHandTrackingOptions): Promise<{ left: AbstractMesh, right: AbstractMesh }> {
         return new Promise(async (resolve) => {
             const riggedMeshes: { [handedness: string]: AbstractMesh } = {};
+            // check the cache, defensive

```

Comment:
Looks all good to me but wondering if the cache should be per scene ???

Reply:
XR is scene-bound because of all of the resources. This can only happen if the scene was already disposed, including all of its resources (i.e. - running a playground again)

Reply:
Thanks